### DISCUSSION TITLE
Tapo Hub H200 and DS230 doorbell

### DISCUSSION THREAD

**Discussion by martorchio** [JurajNyiri/HomeAssistant-Tapo-Control#726] [General]
Hello everyone,

I've recently installed the integration and was trying to add my DS230 doorbell (1.1.17 - build 20240731 rel 53511)

The integration works, and I can add the devices (H200 and DS230). However, I don't have access to the live video stream. Is this normal?

Thanks

**Comment by JurajNyiri** [comment]
Yes, video is only supported if the device exposes rtsp stream.

**Comment by claytonjcoleman** [comment]
> Hello everyone,
> 
> I've recently installed the integration and was trying to add my DS230 doorbell (1.1.17 - build 20240731 rel 53511)
> 
> The integration works, and I can add the devices (H200 and DS230). However, I don't have access to the live video stream. Is this normal?
> 
> Thanks

Did you figure out which entity to use for a "detection" ?
